### DISCUSSION TITLE
Frozen, unresponsive, therefore inoperable

### DISCUSSION THREAD

**Discussion by Andromander** [community/community#50354] [Mobile]
### Select Topic Area

Bug

### Body

I'm unsure when this started, but yesterday, I wanted to view my Issues tab from the home screen. However, upon opening the app, it seems unresponsive to any of my presses. Android does not recognise the app as frozen or crashed, but I'm unable to click anything. Most pages seem affected by this, such as opening via Github URL.

I have deleted Cache and Data, but it has not solved it. If anything, it has now permanently logged me out of that device because the login screen is frozen in this state, meaning now any Github URL is unable to be viewed without uninstalling the app. Similarly, I have uninstalled the app and it has not resolved the issue.

Oppo A52, Android 11, ColorOS V11.1.

**Comment by Mohamed-SayedAlAhl** [comment]
It seems like you are experiencing issues with the Github mobile app on your Oppo A52 running Android 11 with ColorOS V11.1. Since you have already tried deleting the cache and data, as well as uninstalling and reinstalling the app, it's possible that there may be some compatibility issues with the app and your device or OS version.

One thing you can try is to check for any available updates for both the Github app and your device's operating system. Make sure to install any updates that are available to see if this resolves the issue.

If this doesn't work, you can also try contacting Github's support team for further assistance. They may be able to provide additional troubleshooting steps or insights into the issue.

**Comment by Andromander** [comment]
Checking the phone's version, says it's the latest available version. Also reinstalled the app (workaround for navigation issues detailed above) so it should be the latest version. Issue persists.

Something I have noticed: the phone makes use of the whole front as the screen, so the navigation buttons are on the screen, too. When I open the app, I notice that the area where the navigation buttons are remains white, but the login screen (and phone's theme) is dark. Those buttons are the only thing interactive. Not sure what this indicates, but overall, still not fixed.

**Comment by laur89** [comment]
Likely different case, but I stumbled upon this issue some time after having enabled the `Smart Pixels` feature. Reinstall/reboot/cache clear didn't do the trick. Only after disabling smart pixels the app became responsive to touch again.

Perhaps give toggling the SP feature a go and see if that helps.

**Comment by superle3** [comment]
Only some pages were responsive for me when another app (red moon) was overlayed over the whole app. When I turned that overlay off, GitHub mobile became responsive again. But this issue only exists for GitHub so probl not a bug from red moon.

**Comment by MinhKhang273z** [comment]
The system became completely frozen and remained unresponsive to any user input or internal processes. As a result, all normal operations were halted, rendering the system effectively inoperable until it could be restarted or manually recovered.
